projects/plugins/backup/jetpack-backup.php

```diff
@@ -40,16 +40,64 @@
 define( 'JETPACK_BACKUP_PLUGIN_NAME', 'Jetpack Backup' );
 define( 'JETPACK_BACKUP_PLUGIN_URI', 'https://jetpack.com/jetpack-backup' );
 
+// Require Jetpack 10+, if Jetpack is installed.
+if ( defined( 'JETPACK__VERSION' ) && version_compare( '10.0.0', JETPACK__VERSION, '>' ) ) {
```

Comment:
Hmm, this doesn't work because (for me on JN at least), this is loading before the jetpack file is loading, thus `JETPACK__VERSION` is not defined.